cloudinit/distros/__init__.py

```diff
@@ -1039,19 +1045,22 @@ def ensure_sudo_dir(self, path, sudo_base="/etc/sudoers"):
                         "",
                     ]
                     sudoers_contents = "\n".join(lines)
-                    util.write_file(sudo_base, sudoers_contents, 0o440)
+                    util.write_file(admin_sudo_base, sudoers_contents, 0o440)
                 else:
+                    if sudoers_base != admin_sudo_base:
+                        util.write_file(
```

Comment:
Let's log what we've done here, otherwise our write_files logging won't really tell us where we got the contents. "Copying content from /usr/etc/sudoers to /etc/sudoers"